emailapp/models.py

```diff
@@ -15,12 +15,20 @@ def create_superuser(self, email, password=None, **extra_fields):
         extra_fields.setdefault('is_staff', True)
         extra_fields.setdefault('is_superuser', True)
 
-        return self.create_user(email, password, **extra_fields)
+        return self.create_user(email, password,role='Manager', **extra_fields)
 
 class CustomUser(AbstractBaseUser):
+
+    MANAGER = 'Manager'
+    EMPLOYEE = 'Employee'
+    ROLE_CHOICES = [
+        (MANAGER, 'Manager'),
+        (EMPLOYEE, 'Employee'),
+    ]
     email = models.EmailField(unique=True)
     first_name = models.CharField(max_length=30)
     last_name = models.CharField(max_length=30)
+    role = models.CharField(max_length=10, choices=ROLE_CHOICES, default=EMPLOYEE)
     is_active = models.BooleanField(default=True)

```

Comment:
doesn't need to define is_active, is_staff and is_superuser, first_name and last_name.
It is already provided by AbstarctUser.